dev_setup.sh

```diff
@@ -275,24 +279,33 @@ function install_deps() {
     echo "Installing packages..."
     if found_exe zypper ; then
         # OpenSUSE
+        echo ${YELLOW} Installing packages for OpenSUSE...${RESET}
         $SUDO zypper install -y git python3 python3-devel libtool libffi-devel libopenssl-devel autoconf automake bison swig portaudio-devel mpg123 flac curl libicu-devel pkg-config libjpeg-devel libfann-devel python3-curses pulseaudio
         $SUDO zypper install -y -t pattern devel_C_C++
     elif found_exe yum && os_is centos ; then
         # CentOS
+        echo ${YELLOW} Installing packages for Centos...${RESET}
```

Comment:
Maybe BLUE or GREEN?  See comment below.

Reply:
Updated to green, the instructions if no package manager was found is updated to Yellow while highlighting the packages in Blue as previous.